plugins/rocmsmi/rocmsmi

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/perl
+# -*- perl -*-
+
+=head1 NAME
+
+rocmsmi_ - Wildcard plugin for monitoring AMD GPUs
+
+=head1 CONFIGURATION
+
+This is a wildcard plugin.  The plugin link name prefix (after the final "_")
+should be a GPU device numeric ID (0,1,2,3,...).
+
+[rocmsmi_*]
+user root
+env.rocmsmiexec /opt/rocm/bin/rocm-smi
+
+=head1 AUTHOR
+
+Šarūnas Burdulis, sarunas@mail.saabnet.com
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 BUGS
+
+=head1 MAGIC MARKERS
+
+ #%# family=contrib
+
+=cut
+
+use strict;
+use warnings;
+
+my $rocmsmi = defined $ENV{rocmsmiexec} ? $ENV{rocmsmiexec} : '/opt/rocm/bin/rocm-smi';
+my $output;
+my $gpu_id;
+my $temp;
+my $power;
+my $fan;
+my $gpuclock;
+my $memclock;
+my $gpuident;
+
+# GPU ID defined?
```

Comment:
How about adding a bit of convenience here?

There are two options:
A) provide the `suggest` capability (by detecting available cards)
B) supply graphs for all graphic cards automatically via `multigraph`

Option (B) is probably the preferable solution.